operations/mimir-mixin-tools/serve/provisioning-datasources.yaml

```diff
@@ -13,6 +13,13 @@ datasources:
       basicAuthPassword: $DATASOURCE_PASSWORD
     version: 1
     editable: true
+  - name: Mimir (ingest-storage local dev env)
```

Comment:
Note to reviewers: I find it useful to test dashboard changes. I suggest to keep it here.

Reply:
Me too! But it requires running mixin-serve tool with extra parameters:

```
./operations/mimir-mixin-tools/serve/run.sh --docker-network mimir-ingest-storage_default
```